src/Nevermind/Nevermind.Core/Hex.cs

```diff
@@ -60,6 +60,11 @@ public bool Equals(Hex obj)
             return false;
         }
 
+        public byte[] ToBytes()
+        {
```

Comment:
there is implicit conversion operator, not sure if we need it ever

Reply:
I am not a great fan of the Hex class any more by the way, we will review when alpha version fiinished